commands.go

```diff
@@ -450,6 +450,8 @@ type Cmdable interface {
 	GeoSearchStore(ctx context.Context, key, store string, q *GeoSearchStoreQuery) *IntCmd
 	GeoDist(ctx context.Context, key string, member1, member2, unit string) *FloatCmd
 	GeoHash(ctx context.Context, key string, members ...string) *StringSliceCmd
+
+	AclDryRun(ctx context.Context, username, command string, args ...interface{}) *StringCmd
```

Comment:
// or args or commands
should: ACLDryRun(ctx, username, command ...interface)

for us, there is no need to distinguish between command and args.  It's ACL, not Acl, just like CNCF and not Cncf.

Reply:
comment incorporated